README.md

```diff
@@ -2467,6 +2473,28 @@ providing smooth integration with other Consul tools (like `consul maint` or
 `consul lock`). Additionally, exposing these environment variables gives power
 users the ability to further customize their command script.
 
+#### Multiple Commands
+
+The command configured for running on template rendering must be a single
+command. That is you cannot join multiple commands with `&&`, `;`, `|`, etc.
+This is a restriction of how they are executed. **However** you are able to do
+this by combining the multiple commands in an explicit shell command using `sh
+-c`. This is probably best explained by example.
+
+Say you have a couple scripts you need to run when a template is rendered,
+`/opt/foo` and `/opt/bar`, and you only want `/opt/bar` to run if `/opt/foo` is
+successful. You can do that with the command...
+
+`command = "sh -c '/opt/foo && /opt/bar'"
+
+As this is a full shell command you can even use conditionals. So accomplishes the same thing.
+
+`command = "sh -c 'if /opt/foo; then /opt/bar ; fi`"
```

Comment:
I believe this should be

`command = "sh -c 'if /opt/foo; then /opt/bar ; fi'"`

Reply:
Syntax highlighters have broken me. Fixed this as well.